Makefile

```diff
@@ -1,10 +1,17 @@
-setup:
-	curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer | bash
-	pyenv install '2.7.14'
-	pyenv local 2.7.14
-	pyenv virtualenv 2.7.14 samtranslator27
-	pyenv activate samtranslator27
-	pyenv local samtranslator27
+PYENV := $(shell command -v pyenv 2> /dev/null)
+
+install:
+	curl -L https://github.com/pyenv/pyenv-installer/raw/master/bin/pyenv-installer 2> /dev/null | bash
+
+setup: install
+ifndef PYENV
```

Comment:
Will this exit the command at all? If not should we be? We know that it isn't installed and therefore can't use pyenv at the moment.

Reply:
Interesting point now it will install it on the first attempt and then report that it's not configured. I was working on a implementation to get installed properly on my mac but that makes the Makefile complex. Shall i remove the installation and just message that pyenv should be installed properly before you can use it?